src/core/tailer/tailer.go

```diff
@@ -20,7 +20,7 @@ import (
 
 var tailConfig = tail.Config{
 	Follow:    true,
-	ReOpen:    true,
+	ReOpen:    false,
```

Comment:
I think we want to keep this as true in case the log files get rotated.

Reply:
this causes a memory leak...